kolibri/tasks/api.py

```diff
@@ -185,11 +206,7 @@ def _networkimport(channel_id, update_progress=None, check_for_cancel=None):
             update_progress=update_progress,
             check_for_cancel=check_for_cancel)
     except UserCancelledError:
-        try:
-            os.remove(get_content_database_file_path(channel_id))
-        except OSError:
-            pass
-        # TODO: Delete channel data
+        call_command("deletechannel", channel_id, update_progress=update_progress)
```

Comment:
Could the string `"deletechannel"` here and line 219 be replaced with the `DELETE_CHANNEL` constant above?

Reply:
In this case, no - it's the name of the django management command.

Might be nice to clean this up in the future, by using explicit imports for our `call_command` invocations. https://docs.djangoproject.com/en/1.11/ref/django-admin/#running-management-commands-from-your-code